.github/workflows/conda_and_docs_build.yml

```
@@ -33,92 +31,76 @@ on:
     - 'scripts/**'
     - 'NOTICE.txt'
     - 'README.md'
-
   pull_request:
-    branches: [ master ]
+    branches: [master]
     paths-ignore:
     - 'CHANGELOG.md'
     - 'CITATION.cff'
     - 'LICENSE'
     - 'scripts/**'
     - 'NOTICE.txt'
     - 'README.md'
-
 jobs:
   conda_build:
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v3.1.0
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 0
     - name: conda-build
       uses: paskino/conda-package-publish-action@v1.4.4
       with:
-        subDir: 'recipe'
-        channels: '-c conda-forge -c intel -c astra-toolbox -c ccpi'
+        subDir: recipe
+        channels: -c conda-forge -c intel -c astra-toolbox -c ccpi
```

we don't need this channel anymore

```suggestion
        channels: -c conda-forge -c intel -c ccpi
```